src/test/kotlin/uk/gov/dluhc/printapi/factory/UrlFactoryTest.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.dluhc.printapi.factory
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.catchThrowableOfType
+import org.junit.jupiter.api.Test
+import uk.gov.dluhc.printapi.dto.SourceType
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidEroId
+import uk.gov.dluhc.printapi.testsupport.testdata.aValidSourceReference
+
+class UrlFactoryTest {
+
+    private val factory = UrlFactory("http://localhost:8080")
+
+    @Test
+    fun `should create photo URL given source type ANONYMOUS_ELECTOR_DOCUMENTED`() {
```

Comment:
Typo: should be `ANONYMOUS_ELECTOR_DOCUMENT` . Feel free to address this in next PR

Reply:
👍 Good spot - yes, I'll correct that